marquez_client/marquez.py

```diff
@@ -8,35 +8,37 @@
 
 class MarquezClient(object):
     API_PATH = "api/v1"
-    MARQUEZ_HOST_KEY = "MQZ_HOST"
-    MARQUEZ_PORT_KEY = "MQZ_PORT"
+    MARQUEZ_HOST_KEY = "MARQUEZ_HOST"
+    MARQUEZ_PORT_KEY = "MARQUEZ_PORT"
     c = None
     namespace = None
     dataset_api_client = None
     jobs_api_client = None
     namespace_api_client = None
 
-    def __init__(self):
-        try:
-            host_from_configs = os.environ['MQZ_HOST']
-            port_from_configs = os.environ['MQZ_PORT']
-        except KeyError as ke:
-            msg = ("Please provide proper env vars in context: "
-                   "MQZ_HOSTNAME and MQZ_PORT. Missing " + str(ke.args))
-            raise Exception(msg)
-
-        logging.info("Connecting to Marquez at %s:%s",
-                     host_from_configs, port_from_configs)
+    def __init__(self, host=None, port=None, timeout=None):
+        self.host = os.environ.get('MARQUEZ_HOST') or host
```

Comment:
We can use `MARQUEZ_HOST_KEY` instead of inlining it.

Reply:
I feel we should actually remove the use of MARQUEZ_HOST_KEY, it's being used in basically one place.

Reply:
Sure - up to you :)